README.md

```diff
@@ -19,3 +19,19 @@ If you find an issue with JAWS support for ARIA, CSS, HTML, or SVG, please [file
 * Test case that demonstrates the issue - provide linked test case codepen/JSbin etc.
 
 Note: please do not file general support issues here. Please [contact Freedom Scientific](https://www.freedomscientific.com/About/ContactUs) to report general issues with JAWS.
+
+## Triaging Issues
+
+Workflow is as follows:
+
+1. Find an unassigned issue. [Here is a good set of filters to start with](https://github.com/FreedomScientific/VFO-standards-support/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+-label%3Abug+-label%3Aarchive+no%3Aassignee+-label%3A%22Not+a+JAWS+bug%22+-label%3A%22Feature+Request%22+-label%3A%22JAWS+bug+filed%22)
+2. Once you identify an issue to work on, make sure the author followed the guidelines provided above. If not, tag them in a comment asking for more info and tag the ticket with the "More Info Needed" label.
+3. Assign the ticket to yourself and start investigating.
+
+### Resolution
+
+* Issue is fixed in the most recent version of JAWS: tag ticket with "Fixed in JAWS 20XX" label and close
+* Issue is valid in most recent version of JAWS: document your research in a comment and assign issue to Benilda Key.
+* Can't reproduce: If needed, ask more questions of the issue author. You can also reach out to Benilda Key with clarification questions. When you're confident it's the answer, tag with "Can't Reproduce" label and close.
+* Issue appears to be browser specific, not JAWS related: document your research in a comment, tag ticket with appropriate browser issue label (i.e. "Chrome Bug") and close.
```

Comment:
Ideally file a bug on the browser link to this https://web.dev/how-to-file-a-good-bug/